media-gfx/kuickshow/kuickshow-0.11.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_HANDBOOK="forceoptional"
+KFMIN=6.0.0
+QTMIN=6.0.0
+inherit ecm kde.org
+
+DESCRIPTION="Image viewer based on KDE Frameworks"
+HOMEPAGE="https://userbase.kde.org/KuickShow"
+SRC_URI="mirror://kde/unstable/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="6"
```

Comment:
We have generally been moving to slot 0 for leaf packages (not libraries) instead.

Reply:
I only copied the previous ebuild and adjusted the values, assuming they meant that only one of them can be installed. I don't know where that slot-definition originally came from.
But I've set it to 0 and removed the RDEPEND at the end. It seems to build.

Reply:
Yes. We're correcting the slotting as we go on these bumps to KF6/Qt6 based leaf packages.

The RDEPEND blocker was fine and will need to be kept around ~2 years to avoid file collisions against the old SLOT 5 version for slow upgraders.

Reply:
I've added the RDEPEND with :5 again